### PR TITLE
Fix/task3

### PR DESCRIPTION
**Pull request by Dalpatov**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,21 @@
 import React, {useState} from 'react';
```
Comment (usatovaDasha) [inline src/App.js]: space!!!!!!!!!

File: src/App.js [hunk 2]
```diff
@@ -1,25 +1,21 @@
...
-      <Route path='/autorization' component={AutorizationPage}/>
-    </Switch>
-    <Redirect from='/' to='/registration'/>
-   </div>
+      <Switch>
+        <Route path='/maintab' component={MainTab}/>
+        <Route path='/registration' component={RegistrationPage}/>
+        {/* <Route path='/autorization' component={AutorizationPage}/> */}
+      </Switch>
+      <Redirect from='/' to='/registration'/>
```
Comment (usatovaDasha) [inline src/App.js]: redirect in switch

File: src/Components/Alert/Alert.js [hunk 3]
```diff
@@ -2,23 +2,25 @@ import React from 'react';
...
-    setAlertFlag(false);
+    setCheck(false);
   };
 
   return (
     <div >
-      <Snackbar open={state} autoHideDuration={6000} onClose={handleClose}>
-        <Alert onClose={handleClose} severity={alertStyle}>
+      <Snackbar open={state}
+      autoHideDuration={2000}
```
Comment (usatovaDasha) [inline src/Components/Alert/Alert.js:26]: tabs

File: src/Components/Alert/Alert.js [hunk 4]
```diff
@@ -2,23 +2,25 @@ import React from 'react';
...
+    setCheck(false);
   };
 
   return (
     <div >
-      <Snackbar open={state} autoHideDuration={6000} onClose={handleClose}>
-        <Alert onClose={handleClose} severity={alertStyle}>
+      <Snackbar open={state}
+      autoHideDuration={2000} 
+      onClose={handleClose}>
```
Comment (usatovaDasha) [inline src/Components/Alert/Alert.js:27]: tabs

File: src/Components/Alert/Alert.js [hunk 5]
```diff
@@ -2,23 +2,25 @@ import React from 'react';
...
+        <Alert onClose={handleClose} 
+        severity={alertStyle}>
```
Comment (usatovaDasha) [inline src/Components/Alert/Alert.js:30]: tabs

File: src/Components/HeaderAll/HeaderAll.js [hunk 6]
```diff
@@ -1,7 +1,7 @@
-import React, {useState} from 'react';
-import Headpic from '../img/Headpic.svg'
-import '../styles/HeaderAll.css'
-function HeaderAll ({name}){
+import React, { useState } from 'react';
+import Headpic from '../../Img/Headpic.svg';
+import './HeaderAll.css'
```
Comment (usatovaDasha) [inline src/Components/HeaderAll/HeaderAll.js]: missing ;

File: src/Components/InputBox/InputBox.js [hunk 7]
```diff
@@ -0,0 +1,50 @@
+import React, {useState} from 'react';
```
Comment (usatovaDasha) [inline src/Components/InputBox/InputBox.js]: spaces

File: src/Components/InputBox/InputBox.js [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+import {
+  IconButton,
+  InputAdornment,
+  FormControl,
+  OutlinedInput,
```
Comment (usatovaDasha) [inline src/Components/InputBox/InputBox.js]: remove ,

File: src/Components/InputBox/InputBox.js [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+} from '@material-ui/core';
+import Visibility from '@material-ui/icons/Visibility';
+import VisibilityOff from '@material-ui/icons/VisibilityOff';
+
+function InputAdornments({ placeholder, setPassword, password }) {
+
```
Comment (usatovaDasha) [inline src/Components/InputBox/InputBox.js]: remove empty line

File: src/Components/InputBox/InputBox.js [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+      
+  const handleClickShowPassword = () => {
+    setShowPassword(!showPassword);
+  };
+      
+  const handleMouseDownPassword = (e) => {
+    e.preventDefault();
+  };
+        
+return (
```
Comment (usatovaDasha) [inline src/Components/InputBox/InputBox.js]: tabs in all block

File: src/Components/Registration/RegistrationPage.js [hunk 11]
```diff
@@ -1,58 +1,57 @@
...
 import { useHistory } from 'react-router-dom';
 import axios from 'axios';
-import InputBox from '../Components/InputBox';
-import HeaderAll from '../Components/HeaderAll'
-import Bodypic from '../img/Bodypic.svg';
-import Registr from '../styles/Registr.css';
-import Alert from '../Components/Alert';
+import InputBox from '../InputBox/InputBox';
+import HeaderAll from '../HeaderAll/HeaderAll';
+import Bodypic from '../../Img/bodypic.svg';
```
Comment (usatovaDasha) [inline src/Components/Registration/RegistrationPage.js:6]: imports style

File: src/Components/Registration/RegistrationPage.js [hunk 12]
```diff
@@ -1,58 +1,57 @@
...
+        login,
+        password,
+      }).then(res => {
           setLogin('');
           setPassword('');
           setReppassword('');
           history.push('/maintab');
-       });
+      });
       } catch (e) {
```
Comment (usatovaDasha) [inline src/Components/Registration/RegistrationPage.js]: tabs

File: src/Components/Registration/RegistrationPage.js [hunk 13]
```diff
@@ -1,58 +1,57 @@
...
-        setText('Пользователь уже существует');
+      setText('Пользователь уже существует');
       }
-    }
-    setCheck(true);
-    setAlertStyle('error');
+
```
Comment (usatovaDasha) [inline src/Components/Registration/RegistrationPage.js:57]: remove empty line

File: src/Components/Registration/RegistrationPage.js [hunk 14]
```diff
@@ -76,7 +75,7 @@ return(
             setPassword={setReppassword} 
             password={reppassword}/>
               <div className="UserBox">
-                <button className="newUserButton" onClick={()=> addNewUser()}> Зарегестрироваться</button>         
+                <button className="newUserButton" onClick={()=> addNewUser()}>Зарегестрироваться</button>
```
Comment (usatovaDasha) [inline src/Components/Registration/RegistrationPage.js]: long line
ЗарегИстрироваться

Comment (usatovaDasha) [inline src/Components/Registration/RegistrationPage.js]: space

File: src/App.js (hunk 2 above)
Comment (Dalpatov) [reply src/App.js]: This is not a code, this is a stub.

**Pull request closed without merging**